src/serviceView.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2023 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "serviceView.h"
+
+#include <QDialogButtonBox>
+#include <QDir>
+#include <QFormLayout>
+#include <QLabel>
+#include <QTextEdit>
+#include <QTextOption>
+
+#include <QDialogButtonBox>
+#include <QDir>
+#include <QFormLayout>
+#include <QLabel>
+#include <QTextEdit>
+#include <QTextOption>
+
+ServiceView::ServiceView(QDialog *parent, QIcon icon)
+{
+    m_parent = parent;
+    m_icon = icon;
+}
+
+void ServiceView::displayTable(QString svcCmdOutput)
+{
+
+    m_dialog = new QDialog(m_parent);
```

Comment:
```suggestion
    QDialog dialog = new QDialog(m_parent);
```
This doesn't need to be a class wide variable as the variable isn't used outside this function.